package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "dev": "NODE_ENV=development nodemon --exec babel-node src/index.js",
```

Comment:
Could you install `nodemon` too. I can't see it in the `package.json` file.

Reply:
Sure, let me do that!